spectacles/validators.py

```diff
@@ -239,10 +239,11 @@ def _query(self, mode: str = "batch") -> List[SqlError]:
             for explore in model.explores:
                 if mode == "batch" or (mode == "hybrid" and not explore.queried):
                     logger.debug("Querying one explore at at time")
-                    task = loop.create_task(
-                        self._query_explore(session, model, explore)
-                    )
-                    tasks.append(task)
+                    if explore.dimensions:
```

Comment:
Awesome. Could you bump this up a level to line 240? We want to catch this regardless of which query mode we're in and skip to the next explore in the model. That should do it.